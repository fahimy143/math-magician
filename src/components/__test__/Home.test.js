import React from 'react';
import renderer from 'react-test-renderer';
import Home from '../Home';

it('renders to match the test', () => {
  const tree = renderer.create(<Home />).toJSON();
  expect(tree).toMatchSnapshot();
});
