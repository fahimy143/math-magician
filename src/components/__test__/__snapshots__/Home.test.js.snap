// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders to match the test 1`] = `
<div
  className="home"
>
  <h2>
    Welcome to our page!
  </h2>
  <p>
    Lorem ipsum dolor sit amet consectetur, adipisicing elit. Earum ipsum tempore, ea soluta tenetur expedita dignissimos similique deleniti doloribus fugiat! Necessitatibus, sint doloribus deserunt voluptates impedit cumque alias, aperiam illo dolore explicabo iure culpa distinctio inventore perspiciatis sunt ipsam
  </p>
  <p>
    Lorem ipsum dolor sit amet consectetur, adipisicing elit. Earum ipsum tempore, ea soluta tenetur expedita dignissimos similique deleniti doloribus fugiat! Necessitatibus, sint doloribus deserunt voluptates impedit cumque alias, aperiam illo dolore explicabo iure
  </p>
</div>
`;
